ci-operator/step-registry/assisted/nutanix/setup/configure/assisted-nutanix-setup-configure-ref.yaml

```diff
@@ -9,7 +9,7 @@ ref:
       memory: 500Mi
   credentials:
     - namespace: test-credentials
-      name: nutanix-assisted-installer-ci
```

Comment:
I am not sure we use this secret here so if we are not we can remove it, otherwise we need to keep it as is

Reply:
I didn't go into that level of detail, I just made sure the keys are the same for both Nutanix and vSphere.
Not sure if we should do those changes in this PR